scripts/quests/jeuno/The_Gobbiebag_Part_I.lua

```diff
@@ -0,0 +1,131 @@
+-----------------------------------
+-- The Gobbiebag Part I
+-----------------------------------
+--  Log ID: 3, Quest ID: 27
+--  NPC: Bluffnix:  !pos -43 6 -115 245
+-----------------------------------
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require("scripts/globals/status")
+require("scripts/globals/titles")
+require("scripts/globals/utils")
+require("scripts/globals/items")
+require("scripts/globals/interaction/quest")
+-----------------------------------
+local lowerJeunoID = require("scripts/zones/Lower_Jeuno/IDs")
+-----------------------------------
+
+local GobbieQuest = {}
+
+setmetatable(GobbieQuest, {__index = Quest} )
+GobbieQuest.__index = GobbieQuest
+
+function GobbieQuest:new(params)
+    local quest = Quest:new(xi.quest.log_id.JEUNO, params.questId)
+
+    quest.reward = params.reward
+
+    local bagIncrease = 5
+
+    -- If quest is available or accepted, the corect dialogue ID is the expected pre quest inventory size offset by 1
+    local getPendingDialogueId = function (player)
+        return (player:getContainerSize(xi.inv.INVENTORY) + 1)
+    end
+    -- If quest is completed, the corect dialogue ID is the expected post quest inventory size offset by 1
```

Comment:
Just a styling thing, could you please add a newline between functions?

Reply:
Sure, np, also I think I forgot to add , to the last entry in all of the the param tables.

https://github.com/LandSandBoat/server/commit/f76a3b6724f1c148c37a78f0e25c6d339dd07226#:~:text=title%20%3D%20nil-,%7D,-%7D